test/components/search-results.test.js

```diff
@@ -12,12 +12,14 @@ import sinon from 'sinon';
 
 describe('Component', function () {
   global.dataLayer = [];
+  global.impressionsTimestamp = new Map();
```

Comment:
I'm just checking, and the browser support for `map` is not so good. So... I'm going back to an object. :sweat: